code/src/Components/Form.js

```diff
@@ -0,0 +1,98 @@
+import React, {useState} from 'react'
+import Summary from './Summary'
+import FirstQuestion from './FirstQ'
+import SecondQuestion from './SecondQ'
+import ThirdQuestion from './ThirdQ'
+import FourthQuestion from './FourthQ'
+
+const Form = () => {
+    const [name, setName] = useState('')
+    const [amountOfBands, setAmountOfBands] = useState(false)
+    const [bands, setBands] = useState('Korn')
+    const [musicType, setMusicType] = useState([])
+    
+    //State property to go to next question or back
+    const [section, setSection] = useState(0)
+
+    const nextSection = () => {setSection(section + 1)}
+    const backSection = () => {setSection(section - 1)}
+    const restart = () => {setSection(0)}
+
+
+    // Arrays of objects in radio buttons and dropdown
+    const arrayOfBandNumbers = ["0-1", "2-4", "5 or more"]          
+    const bandOptions = ['Korn', 'Greta van Fleet', 'Red Hot Chili Peppers', 'Foo Fighters', 'Rammstein', ]
+
+    const handleSubmit = (event) => {event.preventDefault()}
+
+
+    //Removing or adding musictype in the array from checked checkboxes
+    const onCheckboxChange = (music) => {
+        if (musicType.includes(music)) {
+            const filteredMusicType = musicType.filter(item => {
+                return item !== music                     //If musictype is included it will remove it if clicked again
```

Comment:
Thanks for writing out the comment, this one was pretty hard to understand for me. Great job making it work!!